raiden/tests/utils/mediation_fees.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+
+from raiden.tests.utils import factories
+from raiden.utils.mediation_fees import prepare_mediation_fee_config
+from raiden.utils.typing import ProportionalFeeAmount
+
+
+@pytest.mark.parametrize("flat_fees", [(42, 21), (43, 21)])
+def test_prepare_mediation_fee_config_flat_fee(flat_fees):
+    cli_flat_fee, expected_channel_flat_fee = flat_fees
```

Comment:
You could move this into the `parametrize` by doing
```python
@pytest.mark.parametrize("cli_flat_fee,expected_channel_flat_fee", [(42, 21), (43, 21)])
def test_prepare_mediation_fee_config_flat_fee(cli_flat_fee, expected_channel_flat_fee):
```
Not necessary, but slightly nicer IMO.